src/app/page.tsx

```diff
@@ -1,5 +1,6 @@
 import Image from 'next/image';
 import styles from './page.module.css';
+import '../styles/reset.css';
```

Comment:
that should be in layout.tsx before import './globals.scss';

Reply:
Indeed it should. I'm on it :)